### DISCUSSION TITLE
Maximum number of wallets

### DISCUSSION THREAD

**Discussion by skeutmeier** [MrNaif2018/bitcart#352] [Q&A]
Hi,

Has anybody tested what the maximum number of (readonly / xpub) wallets are that bitcartcc can handle?

Is it possible to estimate the max number of wallets depending on RAM?
What would be the restrictive factor?

Thanks

**Comment by MrNaif2018** [comment]
I know that for e.g. ETH we had 11k wallets loaded and it uses like 300 megabytes ram, but that's with polling support completely disabled
